src/main/java/com/nerodesk/om/aws/AwsDoc.java

```diff
@@ -158,13 +155,19 @@ public Attributes attributes() throws IOException {
     }
 
     /**
-     * Ocket key.
+     * Ocket.
      * @return Key
      */
     private Ocket ocket() {
-        return this.bucket.ocket(
-            String.format("%s/%s", this.user, this.label)
-        );
+        return this.bucket.ocket(this.ocketKey());
+    }
+
+    /**
+     * Ocket key.
+     * @return Ocket key.
+     */
+    private String ocketKey() {
```

Comment:
@darkled looks like there is no need in this method, as it is used only ones. It can be inlined